examples/hello_milvus.py

```diff
@@ -1,90 +1,150 @@
-# Copyright (C) 2019-2020 Zilliz. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
-# with the License. You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software distributed under the License
-# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
-# or implied. See the License for the specific language governing permissions and limitations under the License.
-
+# hello_milvus.py shows the usage of PyMilvus basic methods.
+# 1. connect to Milvus
+# 2. create collection
+# 3. insert entites
```

Comment:
`entities`
Or rather `data`.
I believe that `entities` are what data are after being inserted into Milvus.